src/pages/studyView/table/MultiSelectionTable.tsx

```diff
@@ -232,7 +232,12 @@ export class MultiSelectionTable extends React.Component<
                 headerRender: () => {
                     return (
                         <div
-                            style={{ marginLeft: cellMargin }}
+                            style={{
+                                marginLeft: cellMargin,
+                                flexDirection: 'row-reverse',
+                                display: 'flex',
+                                marginRight: cellMargin,
+                            }}
```

Comment:
Do we need this? I don't see any effect of this style on the column header. It looks same as before.